site/src/pages/release-notes/1.9.0.mdx

```diff
@@ -0,0 +1,237 @@
+---
+date: 2021-06-23
+---
+
+## 🌟 New features
+
+- You can now bind [GraphQL Java](https://www.graphql-java.com/) to <type://ServerBuilder> 
```

Comment:
How about: `A new module `armeria-graphql` integrates with GraphQL Java to let you serve GraphQL requests. ?